src/Module.Server/Modes/Dtv/CrpgDtvGameMode.cs

```diff
@@ -126,7 +126,7 @@ public override void StartMultiplayerGame(string scene)
                 notificationsComponent,
                 new WelcomeMessageBehavior(warmupComponent),
 #if CRPG_SERVER
-                new CrpgDtvServer(rewardServer),
+                new CrpgDtvServer(rewardServer, lobbyComponent),
```

Comment:
CrpgDtvServer.cs already instantiates a MissionLobbyComponent from inheriting MissionMultiplayerGameModeBase.

Reply:
Not an instanciation. It just fetches it. But yes since it's already fetched , you can reuse it